app/commands/track/determine_concept_map_layout.rb

```diff
@@ -143,30 +128,35 @@ def adjacencies
       end
 
       def determine_nodes
-        track.concept_exercises.includes(:taught_concepts, :prerequisites).map.with_index do |exercise, idx|
-          Node.new(
-            index: idx,
-            slug: exercise.slug,
-            taught_concepts: exercise.taught_concepts.to_a,
-            prerequisites: exercise.prerequisites.to_a
-          )
-        end.freeze
+        nodes = track.concept_exercises.includes(:taught_concepts, :prerequisites).flat_map do |exercise|
+          exercise.taught_concepts.map do |concept|
+            Node.new(
+              index: nil,
+              slug: concept.slug,
+              name: concept.name,
+              prerequisites: exercise.prerequisites.to_a
```

Comment:
Could we map to `slug` here (I think only the slugs are used)? 

If so, could we do `pluck(:slug)` instead? (I'm not if that will still honour the `includes` above to avoid an n+1 but it would be good if so)

Reply:
Added in https://github.com/exercism/website/pull/885/commits/da2fc93303754ee5a98b1283d8f5fa9fdb8f3280